test/media-query-provider.js

```diff
@@ -92,6 +92,33 @@ describe('<MediaQueryProvider />', () => {
     });
   });
 
+  describe('when specified device width matches mobile screen', () => {
+    let componentWithValues;
+
+    before(() => {
+      const values = {
+        width: 300,
+        type: 'screen',
+      };
+
+      componentWithValues = mount(
+        <MediaQueryProvider values={values}>
+          <p>Test123</p>
+        </MediaQueryProvider>,
+      );
+    });
+
+    it('should include mobile in the media context', () => {
+      const { media } = componentWithValues.node.getChildContext();
```

Comment:
with this test instead of calling get child context directly maybe we could have an integration test that renders a child wrapped in the media hoc gets the props as expected ?

Reply:
Will add